common/src/main/templates/io/netty/util/collection/KObjectMap.template

```diff
@@ -115,9 +62,4 @@ public interface @K@ObjectMap<V> {
      * Gets the values contained in this map.
      */
     V[] values(Class<V> clazz);
```

Comment:
Do we need this anymore?  I wonder if it is better to just use the `Collection<V>` from `Map`? This way you are not forced to allocate the entire structure in memory.